update package.json, CHANGELOG.md and README.md before release (#218)
VictoriaMetrics/victorialogs-datasource: CHANGELOG.md

## tip

> * FEATURE: enable to sing plugin if token is set. See [this PR](https://github.com/VictoriaMetrics/victorialogs-datasource/pull/217).  

Reviewer: ```suggestion
* FEATURE: enable plugin sign procedure for new releases. See [this PR](https://github.com/VictoriaMetrics/victorialogs-datasource/pull/217).  
```
